Bust Config.rb

```diff
@@ -157,24 +151,59 @@
     face_indexes: [3],
   },
 ]
-# Special cheats for some characters
+
+# map from clothed Simon facesets to corresponding topless Simon
+Busty::TOPLESS_SIMON_MAP = {
+  "face002b" => "face002b_topless",
+  "1 Simon dark" => "face002b_topless",
+  "face002b2" => "face002b_topless2",
+  "1 Simon dark2" => "face002b_topless2",
+  "face002b dark" => "face002b_topless dark",
+  "1 Simon dark eyes" => "face002b_topless dark",
+  "face002b dark2" => "face002b_topless dark2",
+  "1 Simon dark eyes2" => "face002b_topless dark2",
+}
+Busty::TOPLESS_SIMON_MAP.values.uniq.each do |value|
+  Busty::FACE_TO_BUST[value] = "Simon topless"
+end
+# young Simon does not support disrobing
+Busty::SIMON_YOUNG_FACES = ["face002b_topless","face002b_topless dark"]
+
+# Automatically swap certain facesets depending on certain conditions.
+# NB this comes before face-to-bust mapping in character_from_face.
 class Window_Message < Window_Base
   def bust_face_name
+    # if Simon is using his almost-naked sprite, override with topless faceset, unless it's young Simon
+    topless_simon_face = Busty::TOPLESS_SIMON_MAP[$game_message.face_name]
+    if topless_simon_face && $game_actors[2].character_index == 2 && 
+        ($game_message.face_index != 7 || !Busty::SIMON_YOUNG_FACES.include?(topless_simon_face))
```

Comment:
The condition seems weird to me as, as written, it seems it will fail on the perfectly fine `1 Simon dark eyes` eight face?

Maybe something more straightforward like?
```rb
young_simon = ["face002b", "face002b dark"].include?($game_message.face_name) && $game_message.face_index == 7
```

Reply:
That if statement translates to "If we have a topless version of the faceset being shown, and Simon's character sprite is naked, and the face being displayed isn't young Simon, then swap the face", where "the face being displayed isn't young Simon" is true if the face index isn't 7 or the face isn't one of the ones with Young Simon. I guess I could invert that last bit to make it less confusing.

Reply:
Done.